pkg/boot/universalpayload/universalpayload_test.go

```diff
@@ -91,11 +96,13 @@ address sizes	: 39 bits physical, 48 bits virtual
 }
 
 func TestAppendMemMapHOB(t *testing.T) {
-	defer func(old func() (kexec.MemoryMap, error)) { kexecMemoryMapFromSysfsMemmap = old }(kexecMemoryMapFromSysfsMemmap)
-	kexecMemoryMapFromSysfsMemmap = mockKexecMemoryMapFromSysfsMemmap
+	defer func(old func() (kexec.MemoryMap, error)) { kexecMemoryMapFromIOMem = old }(kexecMemoryMapFromIOMem)
+	kexecMemoryMapFromIOMem = mockKexecMemoryMapFromIOMem
+
+	ioMem, err := kexecMemoryMapFromIOMem()
 	hobBuf := &bytes.Buffer{}
 	var hobLen uint64
-	err := appendMemMapHOB(hobBuf, &hobLen)
+	err = appendMemMapHOB(hobBuf, &hobLen, ioMem)
 	if err != nil {
 		log.Fatal(err)
 	}
```

Comment:
nit: simplify return
```suggestion
	if err := appendMemMapHOB(hobBuf, &hobLen, ioMem); err != nil {
		log.Fatal(err)
	}
```